README.md

```diff
@@ -4,15 +4,34 @@
 
 First, install [.NET 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
 
+Second, install clang 
+   <img src="images/vs-clang-install.png" alt="Logo" width="512" height="512">
+
+or via LLVM
```

Comment:
I think we should link to c2cs documentation for this step rather than including in ours (if it is properly documented there)

Reply:
I've fixed this in the documentation I haven't pushed it yet, there is still one slight issue though. Even after doing their documentation part and following the instructions you'll still have to either add clang to path or install the .dll directly into the dotnet tools folder

Reply:
Again sorry for the late reply - wanted to do it on pc not mobile :P

Reply:
AS for it being properly documented, I would say not so much.